include/psa/crypto.h

```diff
@@ -1607,7 +1593,8 @@ psa_status_t psa_mac_abort(psa_mac_operation_t *operation);
 /** Encrypt a message using a symmetric cipher.
  *
  * This function encrypts a message with a random IV (initialization
- * vector).
+ * vector). Use the multipart #psa_cipher_operation_t object to provide
```

Comment:
“the multipart object” doesn't make sense: which multipart object? Maybe something like

> Use the multipart operation interface with a #psa_cipher_operation_t object